src/utils/generateRandomId.js

```diff
@@ -0,0 +1,9 @@
+export const generateRandomId = () => {
+  const rnds = new Uint16Array(5)
```

Comment:
Just curious, does `rnds` stand for 'randoms' or something?

Reply:
yeah, it probably stands for 'randoms'... I stole this code from https://github.com/wikimedia/mediawiki/blob/master/resources/src/mediawiki.user.js#L47